### ISSUE TITLE
Unhelpful completion for 'alias='

### ISSUE THREAD

**Issue by rcorre** [scop/bash-completion#146]
When I type `alias foo=<TAB>` I get the following:

```
alias foo=alias lack='ack --pager='\''less -r'\'''
alias ledger='hledger'
alias ls='ls --color=auto'
alias op='xdg-open'
alias vim='nvim'
alias vims='nvim -S'
... more aliases follow ...
```

This doesn't seem useful to me. If nothing else, it would be nice to have regular filename completion for `alias=`, so I could type

```
alias swagger=~/go/bin/sw<TAB>
```

and end up with

```
alias swagger=~/go/bin/swagger
```

**Comment by scop** [comment]
2f21940 fixes the original intent of offering the existing aliased value if one exists.

It doesn't do what you describe yet, and besides in my opinion just a regular filename completion wouldn't be useful: we'd want executables only, and also other commands from `PATH`.

**Comment by rcorre** [comment]
Thanks! 

> we'd want executables only, and also other commands from PATH

Assuming it would complete paths to executables that aren't in `PATH`, that would work for me. When I use `alias` from the CLI, it is often to get temporary easy access to an exectuable that isn't on my `PATH`.

**Comment by scop** [comment]
Yes, that's how I'd like to see it behave. I took a very brief look at it, but didn't get it to work immediately and ran out of time for now (something using `compgen -c` and probably `-P aliasname=` should do it).

**Comment by rcorre** [comment]
Thanks! I'll try to take a look, I'd like to learn a bit about bash completion

**Comment by scop** [comment]
Great, thanks! Be sure to look into adding unit tests for stuff you work on as well.
